data/modules.js

```diff
@@ -0,0 +1,24 @@
+const modules = [
```

Comment:
Where & why is the modules array being used ?

Reply:
In the `addChapters` utility function to seed the  database.